### PR TITLE
Disable component alerts

### PR DESCRIPTION
**Pull request by johnmcollier**
- Comments out the Component alerts until https://issues.redhat.com/browse/DEVHAS-641 has been completed and we've had a chance to re-evaluate their position within HAS

### CODE REVIEW

**Comment by avi-biton** [comment]
We should also remove the tests for these alerts from https://github.com/redhat-appstudio/o11y/tree/main/test/promql/tests/data_plane

**Comment by johnmcollier** [comment]
@avi-biton Thanks, yup. Just realized I missed those too, so I've updated the PR to remove them

**Review by amisstea — APPROVED** [review]
LGTM

**Comment by avi-biton** [comment]
I think it will fail as we are trying to refer to the alerts in our make file and it is not a `PrometheusRule` object.
It might be that we will need to remove it completely

**Review by yftacherzog — CHANGES_REQUESTED** [review]

File: rhobs/alerting/data_plane/prometheus.component_alerts.yaml
[the remark this answers is not in the document]
Comment (avi-biton) [reply rhobs/alerting/data_plane/prometheus.component_alerts.yaml:1]: @johnmcollier 
Indeed, tested locally and failed.
These files has to be removed completely

[the remark this answers is not in the document]
Comment (johnmcollier) [reply rhobs/alerting/data_plane/prometheus.component_alerts.yaml:1]: Ah, that's too bad. Was hoping we could just comment them out.

In that case, I'll update the PR to remove them

**Comment by avi-biton** [comment]
lgtm

**Review by avi-biton — APPROVED** [review]

**Review by yftacherzog — APPROVED** [review]

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**